components/information-modal.html

```diff
@@ -0,0 +1,23 @@
+<div class="flex flex-col justify-center min-h-screen pt-4 bg-gray-100 min-w-screen justify-center items-center">
+
+    <div class="rounded-lg shadow-lg bg-white my-3 max-w-sm overflow-x-auto">
+        <div class="flex border-b border-gray-100 px-5 py-4">
```

Comment:
Just to make the border a little bit more visible to the eye.

```suggestion
        <div class="flex border-b border-gray-200 px-5 py-4">
```